src/Linux/Host/Broker/Impl/Services/LinuxRHostProcessService.cs

```diff
@@ -41,15 +45,37 @@ class LinuxRHostProcessService : IRHostProcessService {
             return new UnixProcess(process);
         }
 
+        private string GetUsrLibPath() {
+            const string x64Lib = "/usr/lib/x86_64-linux-gnu";
```

Comment:
Ah, I see. The host itself now accepts the argument which is path to R install, right? I think it can do `putenv` to set `LD_LIBRARY_PATH` variable inside the host, immediately before `dlopen`, and it should respect that. That way we don't have to specify this path in two different places (arguments and environment) when launching the host.